### ISSUE TITLE
We should highlight a plugin as a good example to work with

### ISSUE THREAD

**Issue by nickcharlton** [thoughtbot/administrate#1601]
In #1535, @sedubois writes:

> - various users (such as myself) might have functional custom administrate fields in their own private systems but have no clue how to turn that into a gem (although I'm not a good example because I could just have forked the outdated one; but then what should I put on the wiki page? Keep the outdated one, replace with my own, keep both?);

Perhaps [`administrate-field-nested_has_many`][1] is a solution to this problem?

In addressing #1600, we could point to this as a good example to look at.

[1]: https://github.com/nickcharlton/administrate-field-nested_has_many

**Comment by sedubois** [comment]
@nickcharlton yes `administrate-field-nested_has_many` looks like a good start, it also has decent test coverage include a dummy app. One objection might be that it makes use of some gems which are not Rails recommended/standards e.g rspec-rails, factory-bot, etc instead of minitest and fixtures. Anyways we can always start with it then improve.

**Comment by nickcharlton** [comment]
Ah yeah. That'll always be a bit of a fight with a thoughtbot project as [we always use those as a default](https://github.com/thoughtbot/suspenders).

I think as the directory of plugins gets larger, enough people will copy the patterns in their own preferred approaches and so there will be something to link to.

**Comment by sedubois** [comment]
@nickcharlton I hope not to be going too much off topic, but out of curiosity may I ask what are the advantages of using rspec in a new Rails project over Rails' built-in minitest setup (with `ApplicationSystemTestCase` that came in Rails 5.1 and other features such as parallelisation)?

**Comment by nickcharlton** [comment]
It is a bit off-topic, but some thoughts: I think there's a few points in there: I've generally found RSpec to be nicer to work with and more expressive in it's syntax. As for system tests, I'd expect to (gradually) move over to using that (instead of `feature do…` syntax) as it's in Rails itself (e.g.: #1473).
